pkg/addons/default/assets/coredns-1.16.json

```diff
@@ -0,0 +1,358 @@
+{
```

Comment:
can you format this file as well ? the indentation is not consistent (e.g. first level is having more space)

Reply:
just curious how you can prepare this file ?

Reply:
> can you format this file as well ? the indentation is not consistent (e.g. first level is having more space)

sure , i updated the indentation 

The file was copy from the previous 1.15 version 
You can use something like this 
```
kubectl convert -f pod.yaml --local -o json
```